mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/rg-template.json

```diff
@@ -0,0 +1,346 @@
+                {
+                    "name": "default",
+                    "properties": {
+                        "addressPrefix": "10.0.0.0/24",
+                        "serviceEndpoints": [
+                            {
+                                "service": "Microsoft.Storage",
+                                "locations": "[parameters('storageAccountLocations')]"
+                            },
+                            {
+                                "service": "Microsoft.KeyVault",
+                                "locations": ["[parameters('vmLocation')]"]
+                            }
+                        ],
+                        "networkSecurityGroup": {
+                            "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    {
+        "name": "[variables('networkInterfaceName')]",
+        "type": "Microsoft.Network/networkInterfaces",
+        "apiVersion": "2020-11-01",
+        "location": "[parameters('vmLocation')]",
+        "dependsOn": [
+            "[resourceId('Microsoft.Network/publicIPAddresses', variables('publicIpName'))]",
+            "[resourceId('Microsoft.Network/virtualNetworks', variables('vnetName'))]",
+            "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
+        ],
+        "tags": {
+            "displayName": "[variables('networkInterfaceName')]"
+        },
+        "properties": {
+            "ipConfigurations": [
+                {
+                    "name": "ipConfig1",
+                    "properties": {
+                        "privateIPAllocationMethod": "Dynamic",
+                        "publicIPAddress": {
+                            "id": "[resourceId('Microsoft.Network/publicIPAddresses', variables('publicIpName'))]"
+                        },
+                        "subnet": {
+                            "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vnetName'), 'default')]"
+                        }
+                    }
+                }
+            ],
+            "networkSecurityGroup": {
+                "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
+            }
+        }
+    },
+    {
+        "name": "[variables('kvName')]",
+        "type": "Microsoft.KeyVault/vaults",
+        "apiVersion": "2019-09-01",
+        "location": "[parameters('vmLocation')]",
+        "dependsOn": [
+            "[resourceId('Microsoft.Network/virtualNetworks', variables('vnetName'))]"
+        ],
+        "tags": {
+            "displayName": "[variables('kvName')]"
+        },
+        "properties": {
+            "enabledForDeployment": true,
+            "enabledForTemplateDeployment": true,
+            "enabledForDiskEncryption": true,
+            "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
Please don't encode tenantIds.

Reply:
Good catch this was from a generated template, changed it to an ARM script to pull the `tenantId` off the subscription.